.github/workflows/self-test.yaml

```diff
@@ -10,11 +10,15 @@ jobs:
       matrix:
         SDK:
           - web5-js
-          - web5-kt
+          # - web5-kt
```

Comment:
Did you mean to comment this out?

Reply:
i did: it was failing for me, i assumed web5-kt had changed

Reply:
looks like it failed in the latest commit on `main` as well